modules/nf-core/propr/propd/templates/propd.R

```diff
@@ -63,6 +63,68 @@ read_delim_flexible <- function(file, header = TRUE, row.names = 1, check.names
     return(mat)
 }
 
+#' Extract the values for a single metric and convert it into a genes x genes matrix.
+#'
+#' @param object propd object
+one_metric_df <- function(object){
+    result <- getResults(object)
+    genes <- unique(c(result\$Pair, result\$Partner))
+
+    empty_matrix <- matrix(NA, nrow = length(genes), ncol = length(genes), dimnames = list(genes, genes))
+
+    if (object@active == "theta_d"){
+    metric <- "theta"
+    }else{
+    metric <- object@active
+    }
+
+    for (i in 1:nrow(result)) {
+        row_name <- result\$Pair[i]
+        col_name <- result\$Partner[i]
+        empty_matrix[row_name, col_name] <- result[[metric]][i]
+        empty_matrix[col_name, row_name] <- result[[metric]][i]
+        }
+    return(empty_matrix)
+}
+
+#' Extract the proportionality cutoff for a specified FDR value.
```

Comment:
you mean differential proportionality?

Reply:
Si, la funcion venia de propr y no habia cambiado la descripcion.